Assets/Scripts/ScriptableObjects/Data/FlickData.cs

```diff
@@ -0,0 +1,18 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+[CreateAssetMenu(filename = "Flick Data", menuname = "Flick Data", order = 0)]
```

Comment:
Prefix menuname by the folder structure "Entity/...", like in the EffectData, ItemData or StatusEffectData scripts.

Reply:
Done. Thanks!